modules/ossm-cr-parameters.adoc

```diff
@@ -5,9 +5,9 @@
 [id="ossm-cr-parameters_{context}"]
 = Custom resource parameters
 
-The following examples illustrate use of the supported custom resource parameters for {ProductName} and the tables provide additional information about supported parameters.
+The following examples illustrate use of the `ServiceMeshControlPlane` parameters and the tables provide additional information about supported parameters.
```

Comment:
I'd insert a table here explaining the top-level fields (which are then explained below): `istio`, `threescale`. And then another table showing the first level fields under `istio` - such as `global`, `gateways`, `mixer`, `pilot`, ...

Reply:
@brian-avery This would be a good idea for both community and product docs.

Reply:
Not fully understanding what's being suggested here. Are you suggesting something in line with https://maistra-1-0.maistra.io/docs/installation/installation-options/?

Reply:
Can we discuss this here: https://issues.jboss.org/browse/MAISTRA-881 and cover after GA?